.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,8 @@
+{
+	"name": "Node",
+	"image": "mcr.microsoft.com/devcontainers/base:0",
+
+	"features": {
+		"ghcr.io/devcontainers/features/node:1": {}
+	}
+}
```

Comment:
Was that meant to be pushed too?

Reply:
Wasn't intentional, but this would be suitable for devcontainer environments for this repo (I use this to avoid installing `node` on my OS :laughing: )

Reply:
But if you want to push something in this lines, sholdn't come as a "side" effect, and probably should have some instruction in the README